### ISSUE TITLE
matplotlib 3.6 incompatibility

### ISSUE THREAD

**Issue by sorenbear** [sorenbear/magentropy#8]
Colorbar plotting results in error `Unable to determine Axes to steal space for Colorbar.`

In `_plotting.py`:
```
cbar = ax.get_figure().colorbar(
        mpl.cm.ScalarMappable(norm=colorbar_norm, cmap=cmap),
        **colorbar_kwargs
    )
```

Likely related to https://matplotlib.org/stable/api/prev_api_changes/api_changes_3.6.0.html#pending-deprecation-top-level-cmap-registration-and-access-functions-in-mpl-cm.

Specify matplotlib < 3.6 in magentropy 0.1.5, fix in the future.

**Comment by sorenbear** [comment]
Released 0.1.5, with pandas < 2.0 specified.

**Comment by sorenbear** [comment]
and matplotlib < 3.6 specified.

**Comment by sorenbear** [comment]
matplotlib 3.5 is incompatible with python 3.11. For now, this means magentropy 0.1.5 only works with python 3.9 and 3.10.

**Comment by sorenbear** [comment]
Fixed in 0.1.6
